aptos-move/block-executor/src/scheduler.rs

```diff
@@ -164,14 +162,14 @@ impl ValidationStatus {
 
 pub struct Scheduler {
     /// Number of txns to execute, immutable.
-    num_txns: usize,
+    num_txns: TxnIndex,
```

Comment:
Are these changes needed?

Reply:
oh, these, we noticed that MVHashMap and executor were aliasing the same types, but not even using them everywhere, was a mess. Zekun asked @danielxiangzl to change TxnIndex and Incarnation to u32 in the rolling commit diff and that's when it become apparent.
So while refactoring the MVHashMap also took care of this annoyance. Unfortunately, led to bunch of usize casts because Rust doesn't want to use anything else as an index, but other than that, that's the story.
I can go and remove the changes but would rather not - first it takes latency of weeks to get any change in, second it's mostly mechanical changes that should be easy to skim though - sorry for the trouble though.